users/views.py

```diff
@@ -56,10 +59,39 @@ def UserCreateConfirmView(request, token):
         return redirect('users:signup')
```

Comment:
I'd volunteer to look around and do some updating in a separate PR after this is merged.
E.g. adding `user.is_authenticated` to templates and fixing links.